modules/tinymce/src/themes/silver/demo/html/demo/demo.html

```diff
@@ -1,114 +1,596 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no, maximum-scale=1.0">
-<title>silver-theme Demo Page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no, maximum-scale=1.0">
+  <title>silver-theme Demo Page</title>
+  <style type="text/css">
+    .tox-context-bar-layout-transition-animation {
+      transition: top 1s ease, bottom 1s ease;
+    }
+  </style>
 </head>
+
 <body style="">
-<h2>silver-theme Demo Page</h2>
-<div id='tiny-fixed-container' style="border: 10px solid red"></div>
-<div class='tiny-text'>
+  <h2>silver-theme Demo Page</h2>
```

Comment:
Has this file changed meaningfully, or did you just run an auto-formatter on it?

Reply:
A large table was added to allow demonstration of the context toolbar sticking to things.